File: Iniciante/1010.c
#include <stdio.h>

int main() {

  int A, B, C, D;
  double X, Y;

  scanf("%d %d %lf %d %d %lf", &A, &B, &X, &C, &D, &Y);
  printf("VALOR A PAGAR: R$ %.2f\n", B*X + D*Y);

  return 0;
}

File: Iniciante/1010.php
<?php

$l1 = explode(" ", readline());
$l2 = explode(" ", readline());

$a = $l1[0];
$b = $l1[1];
$c = $l1[2];

$d = $l2[0];
$e = $l2[1];
$f = $l2[2];

echo "VALOR A PAGAR: R$ " .  number_format(($b * $c) + ($e * $f), 2, '.', '') . "\n";

?>